core/modules/query/WhereClause.cc

```diff
@@ -107,122 +105,144 @@ void findColumnRefs(std::shared_ptr<BoolTerm> t, ColumnRef::Vector& vector) {
     }
 }
 
+
+void WhereClause::setRootTerm(std::shared_ptr<LogicalTerm> term) {
```

Comment:
Should the parameter be passed by the `const` reference?
`std::shared_ptr<LogicalTerm> const& term`